app/routers/calendar.py

```diff
@@ -1,32 +1,38 @@
 from http import HTTPStatus
 
-from fastapi import APIRouter, Request
+
+from app.dependencies import get_db, templates
+from app.internal import load_parasha as lp
```

Comment:
Prefer to import the full route - it's hard to understand from the code what you mean by cg/lp :)

Reply:
I updated :)